scene/resources/packed_scene.cpp

```diff
@@ -43,6 +43,11 @@
 #include "scene/main/missing_node.h"
 #include "scene/property_utils.h"
 
+#include "modules/modules_enabled.gen.h" // For GDScript
```

Comment:
This should not happen, we should find a different way to do this without having an include to modules from scene.